src/crud/useWavesurferRegions.tsx

```diff
@@ -25,7 +25,12 @@ export interface INamedRegion {
   name: string;
   regionInfo: IRegions;
 }
-
+export const parseRegions = (regionstr: string) => {
+  var segs = JSON.parse(regionstr);
```

Comment:
Should we check to make sure the argument is a string? if it isn't this will throw an error.